src/mgmt/cli_mgmt.c

```diff
@@ -667,6 +720,64 @@ int dc_mgmt_net_cfg_check(const char *name)
 	return 0;
 }
 
+int
+dc_mgmt_get_iface(char *iface)
+{
+	int cpu;
+	int numa;
+	int i;
+
+	cpu = sched_getcpu();
+	if (cpu < 0) {
+		D_ERROR("sched_getcpu() failed: %d (%s)\n", errno, strerror(errno));
+		return d_errno2der(errno);
+	}
+
+	numa = numa_node_of_cpu(cpu);
+	if (numa < 0) {
+		D_ERROR("numa_node_of_cpu() failed: %d (%s)\n", errno, strerror(errno));
+		return d_errno2der(errno);
+	}
+
+	if (resp_g->n_numa_fabric_interfaces <= 0) {
+		D_ERROR("No fabric interfaces initialized.\n");
+		return -DER_INVAL;
+	}
+
+	for (i = 0; i < resp_g->n_numa_fabric_interfaces; i++) {
+		Mgmt__FabricInterfaces *numa_ifaces = resp_g->numa_fabric_interfaces[i];
+		int                     idx;
+
+		if (numa_ifaces->numa_node != numa)
+			continue;
```

Comment:
On the last patch to these structures, I filled out the array with all NUMA nodes, even those with no interfaces, so you should be able to index directly to the desired node now, e.g. `resp_g->numa_fabric_interfaces[numa]`

Reply:
im not sure this the case..  i tried on boro that has 2 numa nodes, 1 interface and it reports just 1 entry:

NUMA/Fabric list has 1 entries:
         Numa 0: 1 interfaces
                 numa_node = 0
                 interface = ib0
                 domain = ib0
                 provider = ofi+tcp

Reply:
Ah, we only fill in the gaps up to the highest NUMA node _that has a fabric interface associated with it_. That still allows a shortcut: if numa < resp_g->nr_numa_fabric_interfaces, index directly, otherwise numa does not have a local iface.

Reply:
OK let's keep this as is for now then and we can optimize later.
in reality the loop bounds here are like 2-4 numas and 4 interfaces at most per numa, so optimization wise, it's less than negligible :-)